### ISSUE TITLE
Trouble Using Pre-trained Models on non-Demo Data

### ISSUE THREAD

**Issue by ecramer22** [OSchoppe/AIMOS#4]
I'm having trouble using the AIMOS pipeline on provided data. I retrieved the data here: https://dataverse.harvard.edu/file.xhtml?persistentId=doi:10.7910/DVN/LL3C1R/Z6VMBY&version=1.1
And was able to use the AIMOS_prepare_custom_data.py program to slice the .nii file into many .tif files as instructed. However, when I go to run the demo using the newly prepared mice data, I get an error 80% into the AIMOS prediction that says there is a value error that leads to division by zero. I was able to get the demo to run on the two different demo files provided in the NACT file but when trying to use prepared data I continue to get this issue. Do you have any suggestions on how to resolve this issue? Thanks so much!

**Comment by aliparac** [comment]
This is because of normalizing data to the mean value in each slice, when it is zero it gives you such error.
use:
if img.mean() > 0.01 :
            img = TF.normalize(img, mean=[img.mean()], std=[img.std()])
return img
instead of 
            img = TF.normalize(img, mean=[img.mean()], std=[img.std()])
in datafunction.py
